.gitignore

```diff
@@ -1,2 +1,4 @@
 *.pyc
-*.pyo
\ No newline at end of file
+*.pyo
+*~
+\#*
```

Comment:
I might add `__pycache__` if I start working on python code, I usually don't create `#*` files, but added it just to be safe while I edited files (it's not a necessary update).